includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -1467,35 +1467,45 @@ public function ajax_add_to_cart() {
 			define( 'WOOCOMMERCE_CART', true );
 		}
 
-		WC()->shipping->reset_shipping();
+		try {
```

Comment:
I just wrapped the code inside the `try...catch` block, but GitHub thinks I changed more stuff here.